Create postgresql-version-check.md (#74)
percona-platform/portal-doc: docs/advisors/checks/postgresql-version-check.md

> # Provide version information on currently installed version of Postgres

Reviewer: ```suggestion
# PostgreSQL version 
```

## Description

> This check provides information about the installed version of PostgresQL. If a new minor version for the installed version is available, the check will generate a message. Additionally, information about the latest version based on Major version is generated.

Reviewer: ```suggestion
This Advisor checks the version of PostgreSQL currently installed, and reports whether a new minor version is available. Information about the latest version based on Major version is also provided.
```

## Resolution

> You can upgrade your postgres as detailed below.

Reviewer: ```suggestion
Follow the instructions below to upgrade PostreSQL:

```

> To upgrade just the minor version, follow the instructions available here:

Reviewer: ```suggestion
- [Upgrade only the minor version](https://docs.percona.com/postgresql/15/minor-upgrade.html).
```

- [Upgrade only the minor version](https://docs.percona.com/postgresql/15/minor-upgrade.html).
https://docs.percona.com/postgresql/15/minor-upgrade.html

> [Upgrade major versions](https://docs.percona.com/postgresql/15/major-upgrade.html)```

Author: ```suggestion
[Upgrade major versions](https://docs.percona.com/postgresql/15/major-upgrade.html)
```

> To upgrade major versions, follow the instructions available here:

Reviewer: ```suggestion
[Upgrade major versions](https://docs.percona.com/postgresql/15/major-upgrade.html)```

> https://docs.percona.com/postgresql/15/major-upgrade.html

Reviewer: ```suggestion
```
